android/src/main/kotlin/id/oddbit/flutter/facebook_app_events/FacebookAppEventsPlugin.kt

```diff
@@ -120,48 +120,48 @@ class FacebookAppEventsPlugin: FlutterPlugin, MethodCallHandler {
-      parameterBundle?.getString("lastName"),
-      parameterBundle?.getString("phone"),
-      parameterBundle?.getString("dateOfBirth"),
-      parameterBundle?.getString("gender"),
-      parameterBundle?.getString("city"),
-      parameterBundle?.getString("state"),
-      parameterBundle?.getString("zip"),
-      parameterBundle?.getString("country")
-    )
-
-    result.success(null)
-  }
-
-  private fun handleUpdateUserProperties(call: MethodCall, result: Result) {
-    val applicationId = call.argument("applicationId") as? String
-    val parameters = call.argument("parameters") as? Map<String, Object>
-    val parameterBundle = createBundleFromMap(parameters) ?: Bundle()
-
-    val requestCallback = GraphRequest.Callback() {
-      @Override
-      fun onCompleted(response: GraphResponse) {
-        val data = response.getJSONObject()
-        result.success(data)
-      }
-    }
-
-    for (key in parameterBundle.keySet()) {
-      Log.d(logTag, "[updateUserProperties] " + key + ": " + parameterBundle.get(key))
-    }
-
-    if (applicationId == null) AppEventsLogger.updateUserProperties(parameterBundle, requestCallback)
-    else AppEventsLogger.updateUserProperties(parameterBundle, applicationId, requestCallback)
-
-    result.success(null)
-  }
+//  private fun handleSetUserData(call: MethodCall, result: Result) {
+//    val parameters = call.argument("parameters") as? Map<String, Object>
+//    val parameterBundle = createBundleFromMap(parameters)
+//
+//    AppEventsLogger.setUserData(
+//      parameterBundle?.getString("email"),
+//      parameterBundle?.getString("firstName"),
+//      parameterBundle?.getString("lastName"),
+//      parameterBundle?.getString("phone"),
+//      parameterBundle?.getString("dateOfBirth"),
+//      parameterBundle?.getString("gender"),
+//      parameterBundle?.getString("city"),
+//      parameterBundle?.getString("state"),
+//      parameterBundle?.getString("zip"),
+//      parameterBundle?.getString("country")
+//    )
+//
+//    result.success(null)
+//  }
+
+//  private fun handleUpdateUserProperties(call: MethodCall, result: Result) {
+//    val applicationId = call.argument("applicationId") as? String
+//    val parameters = call.argument("parameters") as? Map<String, Object>
+//    val parameterBundle = createBundleFromMap(parameters) ?: Bundle()
+//
+//    val requestCallback = GraphRequest.Callback() {
+//      @Override
+//      fun onCompleted(response: GraphResponse) {
+//        val data = response.getJSONObject()
+//        result.success(data)
+//      }
+//    }
+//
+//    for (key in parameterBundle.keySet()) {
+//      Log.d(logTag, "[updateUserProperties] " + key + ": " + parameterBundle.get(key))
+//    }
+//
+//    if (applicationId == null) AppEventsLogger.updateUserProperties(parameterBundle, requestCallback)
+//    else AppEventsLogger.updateUserProperties(parameterBundle, applicationId, requestCallback)
+//
+//    result.success(null)
+//  }
```

Comment:
Removed code should be deleted. But also, please clarify the removal with link to deprecation in api documentation 🙏